src/loader.js

```diff
@@ -203,11 +204,11 @@ async function loader(content) {
     }
   }
 
-  let isAbsolute = isAbsoluteURL(this.resourcePath);
-
-  const filename = isAbsolute
-    ? this.resourcePath
-    : path.relative(this.rootContext, this.resourcePath);
+  const filename =
+    ABSOLUTE_URL_REGEX.test(this.resourcePath) &&
+    !WINDOWS_PATH_REGEX.test(this.resourcePath)
+      ? this.resourcePath
```

Comment:
@alexander-akait 

No, you didn't understand.
`filename` must always be a relative path, not an absolute one.

Webpack expects the relative path in `node_modules/webpack/lib/asset/AssetGenerator.js:545`

```js
// here occurs error if in one of object the `filename` is absolute
newAssetInfo = mergeAssetInfo(data.get("assetInfo"), newAssetInfo);
```

See here `node_modules/webpack/lib/asset/AssetGenerator.js:216`:
```js
getSourceFileName(module, runtimeTemplate) {
		return makePathsRelative( // <== Return always relative path
			runtimeTemplate.compilation.compiler.context,
			module.matchResource || module.resource,
			runtimeTemplate.compilation.compiler.root
		).replace(/^\.\//, "");
	}
```

Reply:
@alexander-akait  sorry, after update to 4.1.2 the bug is fixed, works fine after rebuilds.
Thank you!